src/h5/gridstack-dd-native.ts

```diff
@@ -24,7 +24,7 @@ export class GridStackDDNative extends GridStackDD {
   public resizable(el: GridItemHTMLElement, opts: DDOpts, key?: DDKey, value?: DDValue): GridStackDDNative {
     this._getDDElements(el).forEach(dEl => {
       if (opts === 'disable' || opts === 'enable') {
-        dEl.ddResizable[opts]();
+        dEl.ddResizable && dEl.ddResizable[opts]();
```

Comment:
hey I don't think this is right as is will not call enable if never created... which it needs to. Also move would need same fix.
if you look at the jquery version I force create for enable, and skip for disable/destroy
if (opts === 'enable') {
      $el.resizable().resizable(opts);
    } else if (opts === 'disable' || opts === 'destroy') {
      if ($el.data('ui-resizable')) { // error to call destroy if not there
        $el.resizable(opts);
      }
    }